output/schema/validation-errors.json

```diff
@@ -77,6 +77,7 @@
         "Request: missing json spec query parameter 'help'",
         "Request: missing json spec query parameter 's'",
         "Request: missing json spec query parameter 'v'",
+        "Request: missing json spec query parameter 'master_timeout'",
```

Comment:
This is a validation issue, not an actual bug. The rest-api-spec now has this parameter and https://github.com/elastic/elasticsearch-specification/pull/3252 added it to the Elasticsearch specification.

Reply:
I opened #3260 to fix it.